test/helpers.bash

```diff
@@ -431,7 +431,8 @@ function cleanup_test() {
 	cleanup_pods
 	stop_crio
 	cleanup_lvm
-	rm -rf "$TESTDIR"
+	# Best effort remove the test dir, as sometime it is still busy
+	rm -rf "$TESTDIR" || true
```

Comment:
I worry we might be hiding a real bug here. Can we track down to when the tests became flaky?